WordPress/src/main/java/org/wordpress/android/ui/main/MainBottomSheetFragment.kt

```diff
@@ -45,6 +46,15 @@ class MainBottomSheetFragment : BottomSheetDialogFragment() {
             (dialog?.content_recycler_view?.adapter as? AddContentAdapter)?.update(it ?: listOf())
         })
 
+        viewModel.showQuickStarInBottomSheet.observe(this, Observer { showQuickStartFocusPoint ->
```

Comment:
I'd actually merge this with the `mainActions` live data and add a `boolean` flag to the `MainActionListItem.CreateAction`. This way you could remove this piece of code and you wouldn't have to use the payload to update the recycler view.